### PR TITLE
chore(eslint): apply eslint on whole sources

### PR DESCRIPTION
**Pull request by murdos**
See [.eslintrc.js](https://github.com/jhipster/jhipster-lite/pull/10015/files#r1632309848)

### CODE REVIEW

File: src/main/java/tech/jhipster/lite/generator/client/react/core/domain/ReactCoreModulesFactory.java [hunk 1]
```diff
@@ -56,7 +56,7 @@ public JHipsterModule buildModule(JHipsterModuleProperties properties) {
         .addScript(scriptKey("build"), scriptCommand("tsc && vite build --emptyOutDir"))
         .addScript(scriptKey("preview"), scriptCommand("vite preview"))
         .addScript(scriptKey("start"), scriptCommand("vite"))
-        .addScript(scriptKey("lint"), scriptCommand("eslint --ext .js,.ts,.tsx src/main/webapp/app/**/*"))
+        .addScript(scriptKey("lint"), scriptCommand("eslint --ext .js,.ts,.tsx src/"))
```
Comment (qmonmert) [inline src/main/java/tech/jhipster/lite/generator/client/react/core/domain/ReactCoreModulesFactory.java:59]:
```suggestion
        .addScript(scriptKey("lint"), scriptCommand("eslint --ext .js,.ts,.tsx src"))
```

File: src/main/java/tech/jhipster/lite/generator/client/vue/core/domain/VueModulesFactory.java [hunk 2]
```diff
@@ -67,7 +67,7 @@ public JHipsterModule buildVueModule(JHipsterModuleProperties properties) {
         .addDevDependency(packageName("sinon"), VUE)
         .addScript(scriptKey("build"), scriptCommand("vue-tsc -p tsconfig.build.json --noEmit && vite build --emptyOutDir"))
         .addScript(scriptKey("dev"), scriptCommand("vite"))
-        .addScript(scriptKey("lint"), scriptCommand("eslint --ext .js,.ts,.vue src/main/webapp/app/**/*"))
+        .addScript(scriptKey("lint"), scriptCommand("eslint --ext .js,.ts,.vue src/"))
```
Comment (qmonmert) [inline src/main/java/tech/jhipster/lite/generator/client/vue/core/domain/VueModulesFactory.java:70]:
```suggestion
        .addScript(scriptKey("lint"), scriptCommand("eslint --ext .js,.ts,.vue src"))
```

File: src/test/java/tech/jhipster/lite/generator/client/react/core/domain/ReactCoreModulesFactoryTest.java [hunk 3]
```diff
@@ -44,7 +44,7 @@ void shouldBuildModuleWithStyle() {
       .containing(nodeScript("build", "tsc && vite build --emptyOutDir"))
       .containing(nodeScript("preview", "vite preview"))
       .containing(nodeScript("start", "vite"))
-      .containing(nodeScript("lint", "eslint --ext .js,.ts,.tsx src/main/webapp/app/**/*"))
+      .containing(nodeScript("lint", "eslint --ext .js,.ts,.tsx src/"))
```
Comment (qmonmert) [inline src/test/java/tech/jhipster/lite/generator/client/react/core/domain/ReactCoreModulesFactoryTest.java:47]:
```suggestion
      .containing(nodeScript("lint", "eslint --ext .js,.ts,.tsx src"))
```

File: src/test/java/tech/jhipster/lite/generator/client/vue/core/domain/VueModulesFactoryTest.java [hunk 4]
```diff
@@ -48,7 +48,7 @@ void shouldCreateVueModule() {
         .containing(nodeDependency("vue-router"))
         .containing(nodeScript("build", "vue-tsc -p tsconfig.build.json --noEmit && vite build --emptyOutDir"))
         .containing(nodeScript("dev", "vite"))
-        .containing(nodeScript("lint", "eslint --ext .js,.ts,.vue src/main/webapp/app/**/*"))
+        .containing(nodeScript("lint", "eslint --ext .js,.ts,.vue src/"))
```
Comment (qmonmert) [inline src/test/java/tech/jhipster/lite/generator/client/vue/core/domain/VueModulesFactoryTest.java:51]:
```suggestion
        .containing(nodeScript("lint", "eslint --ext .js,.ts,.vue src"))
```

File: src/main/java/tech/jhipster/lite/generator/client/react/core/domain/ReactCoreModulesFactory.java (hunk 1 above)
Comment (murdos) [reply src/main/java/tech/jhipster/lite/generator/client/react/core/domain/ReactCoreModulesFactory.java:59]: Why? With the slash it's explicit that this is a directory.

Comment (qmonmert) [reply src/main/java/tech/jhipster/lite/generator/client/react/core/domain/ReactCoreModulesFactory.java:59]: we know that `src` is a directory

Comment (pascalgrimaud) [reply src/main/java/tech/jhipster/lite/generator/client/react/core/domain/ReactCoreModulesFactory.java:59]: if it does the same, let's keep with /, so no ambiguity

**Pull request merged**